src/pid/mountinfo.rs

```diff
@@ -0,0 +1,178 @@
+//! Information about mounts from `/proc/[pid]/mountinfo`.
+
+use std::fs::File;
+use std::io::{BufRead, BufReader, Result};
+use std::path::PathBuf;
+use std::str::{self, FromStr};
+
+use libc::pid_t;
+
+use parsers::{map_result, parse_isize, parse_usize};
+
+/// Process mounts information.
+///
+/// See `man 5 proc` for format details.
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct Mountinfo {
+    /// Unique ID for the mount.
+    pub mount_id: isize,
+    /// ID of the parent mount.
+    pub parent_id: isize,
+    /// Device major ID (class).
+    pub major: usize,
+    /// Device minor ID (instance).
+    pub minor: usize,
+    /// Pathname which forms the root of this mount.
+    pub root: PathBuf,
+    /// Mount pathname relative to the process's root.
+    pub mount_point: PathBuf,
+    /// mount options.
+    pub mount_options: Vec<String>,
```

Comment:
(I guess you refer to section "Filesystem-independent mount options", which I think are  mount/fstab specific options, see user/owner/x-* cases).

I see, but what I mean is that from the outside the split seems a bit arbitrary (as it is based on how internally the kernel stores/prints those), while they are just plain string-options. As much as I'm not a fan of string-typing, from an API-user point of view I don't think this mixed enum/string type provides anything in term of type-safety, and at the same time looks less handy to use.

(This is just to say, I politely disagree with this suggestion and asking for a second thought. But I need to rebase anyway and you are the maintainer, so I can change this one too in the next iteration :)